app/src/main/java/com/infomaniak/swisstransfer/ui/utils/HumanReadableSizeUtils.kt

```diff
@@ -24,10 +24,20 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.pluralStringResource
 import com.infomaniak.swisstransfer.R
+import com.infomaniak.swisstransfer.ui.components.FileUi
 
 object HumanReadableSizeUtils {
-    fun getHumanReadableSize(context: Context, sizeInBytes: Long): String {
-        return Formatter.formatShortFileSize(context, sizeInBytes)
+    private const val TOTAL_FILE_SIZE: Long = 50_000_000_000L
+
+    private fun getHumanReadableSize(context: Context, sizeInBytes: Long) = Formatter.formatFileSize(context, sizeInBytes)
+
+    private fun getFilesSizeInBytes(files: List<FileUi>) = files.sumOf { it.fileSizeInBytes }
+
+    fun Context.getFilesSize(files: List<FileUi>): String = getHumanReadableSize(this, getFilesSizeInBytes(files))
```

Comment:
Maybe we should call it something along the lines of `getSpaceUsed` to match the `getSpaceLeft`? If "FilesSize" is more meaningful than "Space", we can also replace space with file size, what do you prefer?